### PR TITLE
Redesign dashboard

### PR DESCRIPTION
**Pull request by aulonm**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "vuexfire": "^3.2.4"
   },
   "devDependencies": {
-    "@firebase/testing": "^0.20.11",
+    "@firebase/rules-unit-testing": "^1.0.7",
```
Comment (aulonm) [inline package.json:51]: `@firebase/testing` is deprecated, so I changed it to the new dependency `@firebase/rules-unit-testing`

File: src/components/DashboardObjective.vue [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+<script>
+export default {
+  name: 'DashboardObjective',
+
+  components: {
+    DashboardProgressBar: () => import('@/components/DashboardProgressBar.vue'),
+  },
+
+  data: () => ({
```
Comment (kiledal) [inline src/components/DashboardObjective.vue:30]: rm?

File: src/components/DashboardProgressBar.vue [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+
+<script>
+import { scaleLinear, format, select } from 'd3';
+
+export default {
+  name: 'DashboardProgressBar',
+
+  data: () => ({
+    completed: false,
+    darkmode: true,
```
Comment (kiledal) [inline src/components/DashboardProgressBar.vue:27]: remove this?

File: src/views/DashboardHome.vue [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+</template>
+
+<script>
+import { mapState } from 'vuex';
+import PieChart from '@/util/PieChart';
+
+export default {
+  name: 'DashboardHome',
+  data: () => ({
+    team: [],
```
Comment (kiledal) [inline src/views/DashboardHome.vue:46]: remove?

**Review by kiledal — APPROVED** [review]

**Review by kiledal — APPROVED** [review]

**Pull request merged**